editor/configs/emsColumns.tsx

```diff
@@ -38,6 +38,17 @@ export const ALL_EMS_COLUMNS = {
     valueFormatter: formatCrashMatchStatus,
     sortable: true,
   },
+  non_cr3_match_status: {
+    path: "non_cr3_match_status",
+    label: "Non-CR3 match status",
+    valueFormatter: formatCrashMatchStatus,
+    sortable: true,
+  },
+  atd_apd_blueform_case_id: {
+    path: "atd_apd_blueform_case_id",
+    label: "Non-CR3 Case ID",
```

Comment:
Issue said to use this column, but just to double check, we dont want to use the `matched_non_cr3_case_ids` column incase more than one were matched? @johnclary

Reply:
I suppose we could add `matched_non_cr3_case_ids` and `matched_crash_pks` to this table and make them hidden by default. That might be nice to have here?